docker/build/tomato_service/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install --no-instal
 ADD get-pip.py /tmp/get-pip.py
 RUN python /tmp/get-pip.py; rm /tmp/get-pip.py
   
-RUN pip install mongoengine\>=0.10,\<0.11 pymongo\>=3.0,\<3.1 pyopenssl\<0.16 msgpack-python\<0.5 pyyaml\<4
+RUN pip install mongoengine\>=0.10,\<0.11 pymongo\>=3.0,\<3.1 pyopenssl\<17 msgpack-python\<0.5 pyyaml\<4 psutil
```

Comment:
I think you can remove the versionrule for pyopenssl or does it have dependences for psutil?